src/main/scala/com/dwolla/aws/lambda/Handler.scala

```diff
@@ -0,0 +1,53 @@
+package com.dwolla.aws.lambda
+
+import aws.AWS.CloudWatch
+import aws.cloudwatch._
+import aws.lambda._
+import cats.effect._
+import com.dwolla.aws.cloudwatch._
+import com.dwolla.aws.lambda.ScheduledEventStream._
+import fs2._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.language.{higherKinds, postfixOps}
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+import scala.scalajs.js.{JSON, _}
+
+object Handler {
+
+  @JSExportTopLevel("removeCloudWatchAlarms")
+  val handler: ScheduledHandler = (event, _, callback: LambdaCallback[Unit]) ⇒ {
+    Stream.emit(event)
+      .covary[IO]
+      .through(handleScheduledEvents[IO])
+      .compile
+      .drain
+      .unsafeRunAsync {
+        case Left(t) ⇒ callback(js.Error(t.getMessage), js.undefined)
+        case Right(()) ⇒ callback(null, ())
+      }
+  }
+
+  def handleScheduledEvents[F[_]](input: Stream[F, ScheduledEvent])
```

Comment:
I noticed you are calling this a `ScheduledEvent` could you please explain why?  Does it have something to do with the stream processing, NodeJS, or the Autoscaling event the lambda is receiving?

Reply:
That's what AWS calls this event type. The types here are derived from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/50dafbd8e340f232392f5d7b88d163a046ebfdd5/types/aws-lambda/index.d.ts#L325-L337

Reply:
[Here it is](https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ScheduledEvent.java#L25) in the Java Lambda library, too.